scrapy/core/http2/stream.py

```diff
@@ -163,7 +163,7 @@ def receive_window_update(self, delta):
         Arguments:
             delta -- Window change delta
         """
-        if self.remaining_content_length > 0 and not self.stream_closed_local:
+        if self.stream_closed_local is False:
```

Comment:
:lipstick: 

```suggestion
        if not self.stream_closed_local:
```

Unless it can be something else (`None`), which would be a bit confusing.